src/main/java/de/varoplugin/cfw/version/OneSeventeenVersionAdapter.java

```diff
@@ -50,8 +50,10 @@ protected void initRespawn() {
     }
 
     @Override
-    protected void initXp() {
-        this.initXp("net.minecraft.world.entity.player.EntityHuman", "net.minecraft.world.food.FoodMetaData");
+    protected void initXp(Player player) {
+        String entityHumanClass = VersionUtils.getNmsClass() + ".world.entity.player.EntityHuman";
+        String entityExperienceOrbClass = VersionUtils.getNmsClass() + ".world.entity.EntityExperienceOrb";
```

Comment:
I'm pretty sure this is incorrect as VersionUtils#getNmsClass should be `net.minecraft.server` on 1.17

Reply:
Maybe, as I said 1.17 is untested